dom/attr.ts

```diff
@@ -1,16 +1,13 @@
 import {splitStringValue} from "./utils";
 
 const SPECIAL_ATTRIBUTE_SETTERS = /^(html|text|css)$/;
-const customDataStorage = new window.WeakMap();
+const customDataStorage = new (window as any).WeakMap();
```

Comment:
You don't need this anymore after merging #127.
```suggestion
const customDataStorage = new WeakMap();
```